server/controllers/utils/errorMessages.ts

```diff
@@ -96,6 +97,18 @@ export const errorMsgDocumentUpload = {
     )}`,
 }
 
+export const errorMsgNoteFileUpload = {
```

Comment:
`errorMsgDocumentUpload` can be re-used for most of these, apart from the file format one